regulations/management/commands/fetch_wkhtmltox.py

```diff
@@ -0,0 +1,18 @@
+import os
+import subprocess
+
+from django.core.management.base import BaseCommand
+
+
+BIN_PATH = (
+    'http://download.gna.org', 'wkhtmltopdf', '0.12', '0.12.3',
+    'wkhtmltox-0.12.3_linux-generic-amd64.tar.xz',
+)
+
+
+class Command(BaseCommand):
+    help = 'Fetch wkhtmltox binary'
+
+    def handle(self, **options):
+        subprocess.check_call(['wget', '-r', os.path.join(*BIN_PATH)])
+        subprocess.check_call(['tar', 'xzvf', BIN_PATH[-1]])
```

Comment:
This downloads a linux binary, so it's going to be hard to make it portable. I'm changing to error if the user's not running linux so we're explicit about this.